Main.java

```diff
@@ -0,0 +1,64 @@
+class Stack {
+    static final int MAX = 1000;
+    int top;
+    int a[] = new int[MAX]; // Maximum size of Stack
+
+    boolean isEmpty() {
+        // Write your code here
+        if (a.length == 0) {
+            return true;
+        }
+        return false;
+    }
+
+    Stack() {
+        // Initialize your constructor
+    }
+
+    boolean push(int x) {
+        // Check for stack Overflow
+        // Write your code here
+        for (int i = 0; i < MAX; i++) {

```

Comment:
What was the need of iterating over here?

Reply:
Since I am implementing Stack as an array I need to insert/push each element so I am iterating over each index of an array